src/test/unit/mocks/dailyCauseListMetaData.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
youve duplicated files, please remove and use the `returnedArtefacts.json`

Reply:
removed duplicated file and now using returnedArtefacts.json.